content/en/integrations/guide/_index.md

```diff
@@ -28,8 +28,20 @@ private: true
     {{< nextlink href="integrations/guide/aws-terraform-setup" tag="AWS" >}}AWS integration automatic setup with Terraform{{< /nextlink >}}
     {{< nextlink href="integrations/guide/aws-manual-setup" tag="AWS" >}}AWS integration manual setup{{< /nextlink >}}
     {{< nextlink href="integrations/guide/aws-integration-troubleshooting" tag="AWS" >}}Troubleshooting the AWS integration{{< /nextlink >}}
+    {{< nextlink href="integrations/guide/monitor-your-aws-billing-details" tag="AWS" >}}Monitor your AWS billing details{{< /nextlink >}}
+    {{< nextlink href="integrations/guide/aws-integration-and-cloudwatch-faq" tag="AWS" >}}AWS integration and CloudWatch FAQ{{< /nextlink >}}
+    {{< nextlink href="integrations/guide/error-datadog-not-authorized-sts-assume-role" tag="AWS" >}}Error: Datadog is not authorized to perform sts:AssumeRole{{< /nextlink >}}
     {{< nextlink href="integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose" tag="AWS" >}}AWS CloudWatch Metric Streams with Kinesis Data Firehose{{< /nextlink >}}
     {{< nextlink href="integrations/guide/amazon_cloudformation" tag="AWS" >}}Using Amazon CloudFormation{{< /nextlink >}}
+    {{< nextlink href="integrations/guide/cloud-metric-delay" tag="AWS" >}}Cloud metric delay{{< /nextlink >}}
+    {{< nextlink href="integrations/guide/aws-integration-and-cloudwatch-faq" tag="AWS" >}}AWS integration and CloudWatch FAQ{{< /nextlink >}}
```

Comment:
This one got listed twice: here and in line 32. Is there one that got missed?

Reply:
Thanks for catching this! I'm not *totally* sure, but I think this may have been why I had to re-add the AWS manual setup guide link back in this commit:
https://github.com/DataDog/documentation/pull/14599/commits/4e3e9b3b701386e0cc7def0e9afeb586a14b8fc8

Regardless, I've removed the duplicate, and from what I can tell nothing is missing. Thanks again for taking a look!